src/Reflection/ReflectionClass.php

```diff
@@ -1038,12 +1077,20 @@ private function parseTraitUsages(): void
             foreach ($adaptations as $adaptation) {
                 $usedTrait = $adaptation->trait;
                 if ($usedTrait === null) {
-                    $usedTrait = $traitNames[0];
+                    $usedTrait = end($traitNames);
```

Comment:
Did the index change in upstream `nikic/php-parser`?

Reply:
No, we just need to read the latest.